docs/_coverpage.md

```diff
@@ -0,0 +1,10 @@
+<!-- _coverpage.md -->
+
+![logo](_media/flow.svg)
+
+# flow-typed <small>3</small>
```

Comment:
What's the 3 for? meant to represent that we're currently on v3? How about `3.x.x` otherwise I get this feeling it's like a third iteration of flow-typed